index.html

```diff
@@ -1392,10 +1392,10 @@ <h5>
             <p its-locale-filter-list="zh-hans" lang="zh-hans">括号用于行内注释、说明。</p>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">括號用於行內注釋、說明。</p>
 
-            <p its-locale-filter-list="en" lang="en">According to <cite>Punctuation Guidance</cite> (2008 revised edition) issued by the Ministry of Education in Taiwan, parentheses used in Chinese include <span class="uname" translate="no">U+FF08 FULLWIDTH LEFT PARENTHESIS</span> [（], <span class="uname" translate="no">U+FF09 FULLWIDTH RIGHT PARENTHESIS</span> [）] and <span class="uname" translate="no">U+2E3A TWO-EM DASH</span> [⸺] or <span class="uname" translate="no">U+2014 EM DASH</span> [—]. Either one two-em dash (<span class="uname" translate="no">U+2E3A TWO-EM DASH</span> [⸺]) or two consecutive em dashes can be used.</p>
+            <p its-locale-filter-list="en" lang="en">According to <cite>Punctuation Guidance</cite> (2008 revised edition) issued by the Ministry of Education in Taiwan, parentheses used in Chinese include <span class="uname" translate="no">U+FF08 FULLWIDTH LEFT PARENTHESIS</span> [（], <span class="uname" translate="no">U+FF09 FULLWIDTH RIGHT PARENTHESIS</span> [）] and <span class="uname" translate="no">U+2E3A TWO-EM DASH</span> [⸺] or <span class="uname" translate="no">U+2014 EM DASH</span> [—]. In the latter case, either one two-em dash (<span class="uname" translate="no">U+2E3A TWO-EM DASH</span> [⸺]) or two consecutive em dashes can be used.</p>
             <p its-locale-filter-list="zh-hans" lang="zh-hans">台湾教育部的《重订标点符号手册》（2008年修订版）称括号为夹注号，分甲式及乙式，甲式为<span class="uname" translate="no">U+FF08 FULLWIDTH LEFT PARENTHESIS</span> [（]与<span class="uname" translate="no">U+FF09 FULLWIDTH RIGHT PARENTHESIS</span> [）]，乙式则为一对各占二个汉字空间的<span class="uname" translate="no">U+2E3A TWO-EM DASH</span>[⸺]或一组两个<span class="uname" translate="no">U+2014 EM DASH</span>[—]。括号属于夹注符号。</p>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">台灣教育部的《重訂標點符號手冊》（2008年修訂版）稱括號為夾注號，分甲式及乙式，甲式為<span class="uname" translate="no">U+FF08 FULLWIDTH LEFT PARENTHESIS</span> [（]與<span class="uname" translate="no">U+FF09 FULLWIDTH RIGHT PARENTHESIS</span> [）]，乙式則為一對各佔二個漢字空間的<span class="uname" translate="no">U+2E3A TWO-EM DASH</span>[⸺]或一組兩個<span class="uname" translate="no">U+2014 EM DASH</span>[—]。括號屬於夾注符號。</p>
-            <p its-locale-filter-list="en" lang="en"><cite>General Rules for Punctuation</cite> (GB/T 15834–2011), the national standard issued by China Central Government, lists em dash as one type of dash.</p>
+            <p its-locale-filter-list="en" lang="en">The description of parentheses in <cite>General Rules for Punctuation</cite> (GB/T 15834–2011), the national standard issued by China Central Government, are basically the same as above, but it lists em dashes as one type of dash.</p>
```

Comment:
'are basically' -> 'is basically' (refers to description)

Reply:
Fixed. Thanks!